### PR TITLE
Fix ONNX bug, add symbolic for full

### PR DESCRIPTION
**Pull request by apaszke**

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -975,13 +975,24 @@ def zeros_like(g, input):
...
     # NOTE: no way to set device in ONNX, so we ignore it
     return g.op("ConstantFill", shape, dtype_i=scalar_type_to_onnx[scalar_type],
                 input_as_shape_i=1, value_f=0)
 
 
+def full(g, shape, value, scalar_type, layout, device):
+    const_value = _maybe_get_const(value, 't')
+    if _is_value(const_value):
+        tmp = zeros(shape, scalar_type, layout, device)
+        return add(tmp, value, g.op("Constant", value_t=torch.tensor(1)))
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: Can we just use None for `alpha`?

**Review by houseroad — COMMENTED** [review]
Overall, LG, can you add some test cases either in https://github.com/pytorch/pytorch/blob/master/test/onnx/test_operators.py or https://github.com/pytorch/pytorch/blob/master/test/onnx/test_pytorch_onnx_caffe2.py, or both? Thanks

File: torch/onnx/symbolic.py (hunk 1 above)
Comment (apaszke) [reply torch/onnx/symbolic.py]: No, this None serves a different purpose (`Scalar-Tensor` operators don't have it).

**Review by houseroad — APPROVED** [review]
The expect file looks correct, but very complicated.

**Pull request closed without merging**